src/Nethermind/Nethermind.Merge.Plugin/EngineRpcModule.Paris.cs

```diff
@@ -65,17 +65,17 @@ private async Task<ResultWrapper<ForkchoiceUpdatedV1Result>> ForkchoiceUpdated(F
 
     private async Task<ResultWrapper<PayloadStatusV1>> NewPayload(ExecutionPayload executionPayload, int version)
     {
-        bool noGcRegion = GC.TryStartNoGCRegion(10.MB(), true);
-
-        try
+        if (!executionPayload.Validate(_specProvider, version, out string? error))
         {
-            if (!executionPayload.Validate(_specProvider, version, out string? error))
-            {
-                if (_logger.IsWarn) _logger.Warn(error);
-                return ResultWrapper<PayloadStatusV1>.Fail(error, ErrorCodes.InvalidParams);
-            }
+            if (_logger.IsWarn) _logger.Warn(error);
+            return ResultWrapper<PayloadStatusV1>.Fail(error, ErrorCodes.InvalidParams);
+        }
 
-            if (await _locker.WaitAsync(_timeout))
+        if (await _locker.WaitAsync(_timeout))
+        {
+            bool noGcRegion = GC.TryStartNoGCRegion(10.MB(), true);
```

Comment:
In my experience, analogues measure sadly heavily increased spikes amount... suggest to check for that with some 24h test run before merging into master

Reply:
analogue? You mean stopwatch?